docker/files/humble_base.repos

```diff
@@ -1,21 +1,17 @@
 repositories:
-  flatland:
-    type: git
-    url: https://github.com/avidbots/flatland.git
-    version: ros2-humble
-  turtlebot3:
-    type: git
-    url: https://github.com/ROBOTIS-GIT/turtlebot3.git
-    version: humble-devel
   beluga:
     type: git
     url: https://github.com/Ekumen-OS/beluga.git
-    version: a60c5a1102752a2a25f8e38be72dc0560880ccf3
+    version: b6d5ee387cd196aa3c11617c993aa6300c7a9484
   apriltag_ros:
     type: git
     url: https://github.com/Adlink-ROS/apriltag_ros.git
     version: foxy-devel
-  turtlebot3_simulations:
+  gz_ekumen_worlds:
+    type: git
+    url: https://github.com/Ekumen-OS/gz_ekumen_worlds.git
+    version: main
+  nav2_minimal_turtlebot_simulation:
     type: git
-    url: https://github.com/ROBOTIS-GIT/turtlebot3_simulations.git
-    version: humble-devel
+    url: https://github.com/ros-navigation/nav2_minimal_turtlebot_simulation.git
+    version: main
```

Comment:
Same.

Reply:
Done in https://github.com/Ekumen-OS/beluga-demos/pull/11/commits/c932c6d8bbff922bb3c1bff16fed9476c603051e.

Reply:
Done in https://github.com/Ekumen-OS/beluga-demos/pull/11/commits/c932c6d8bbff922bb3c1bff16fed9476c603051e.